.env.example

```
@@ -6,3 +6,5 @@ KAKAO_CLIENT_SECRET=
 NEXT_PUBLIC_AUTH_ROUTE_GUARD_ENABLED=
 NEXT_PUBLIC_GOOGLE_CLIENT_ID=
 NEXT_PUBLIC_KAKAO_CLIENT_ID=
+SENTRY_DSN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SENTRY_DSN key should go before the TEAM_ID key

(UnorderedKey)